zoo-css-challenge/index.html

```diff
@@ -78,6 +79,19 @@ <h3>Height and Size</h3>
 			<p>Giraffes use their height to good advantage and browse on leaves and buds in treetops that few other animals can reach (acacias are a favorite). Even the giraffe's tongue is long! The 21-inch tongue helps them pluck tasty morsels from branches. Giraffes eat most of the time and, like cows, regurgitate food and chew it as cud. A giraffe eats hundreds of pounds of leaves each week and must travel miles to find enough food.</p>
 		</section>
 
+		<section class="horse">
+			<h3>Horses</h3>
+			<p> <img class="image-horse" src="./horses.jpg" alt="Horses">Horses and humans have an ancient relationship. Asian nomads probably domesticated the first horses some 4,000 years ago, and the animals remained essential to many human societies until the advent of the engine. Horses still hold a place of honor in many cultures, often linked to heroic exploits in war.
+				</p>
+				<br>
+				<br>
+				<br>
+				<br>
```

Comment:
```suggestion
```
`<br>` is not really used by frontend developers. Instead, we would use CSS to add margins to create space below the `p` here

Reply:
Great, Thank you very much